Dockerfile

```diff
@@ -0,0 +1,60 @@
+# Define function directory
+ARG FUNCTION_DIR="/function"
+
+# Base image of the docker container
+FROM node:lts-slim as build-image
+
+# Include global arg in this stage of the build
+ARG FUNCTION_DIR
+
+# AWS Lambda runtime dependencies
+RUN apt-get update && \
+    apt-get install -y \
+        g++ \
+        make \
+        unzip \
+        libcurl4-openssl-dev \
+        autoconf \
+        automake \
+        libtool \
+        cmake \
+        python3 \
+        libkrb5-dev
+
+# Copy function code
+RUN mkdir -p ${FUNCTION_DIR}/
+
+COPY opensearch-reporting-cli-1.0.0.tgz ${FUNCTION_DIR}/
+RUN ls ${FUNCTION_DIR}/
+WORKDIR ${FUNCTION_DIR}
+RUN tar -xzf opensearch-reporting-cli-1.0.0.tgz
+RUN mv package/* .
+RUN ls
+RUN npm install
+RUN npm install aws-lambda-ric
+
+# Build Stage 2: Copy Build Stage 1 files in to Stage 2. Install chromium dependencies and chromium.
+FROM node:lts-slim
+# Include global arg in this stage of the build
+ARG FUNCTION_DIR
+# Set working directory to function root directory
+WORKDIR ${FUNCTION_DIR}
+# Copy in the build image dependencies
+COPY --from=build-image ${FUNCTION_DIR} ${FUNCTION_DIR}
+RUN ls ${FUNCTION_DIR}/
```

Comment:
Is this ls required?

Reply:
Not required but helps to see if required files are present before deploying the image (added bcoz it helped me to debug, we can totally remove all `ls` commands)

Reply:
Because it increases the image size and what is the final image size?

Reply:
might be better to remove, i think it adds an unnecessary layer to image

Reply:
removed. I am bit confused about size. On my local it shows 1.16GB but on ECR it shows 357.10MB. 

![image](https://user-images.githubusercontent.com/56703525/216156681-ac13f5e9-e2f0-4dd2-8f59-2890116a2324.png)
![image](https://user-images.githubusercontent.com/56703525/216156782-6128e1b0-0f7a-4c38-a88c-d4561154307f.png)